lib/markbind/lib/parser.js

```diff
@@ -293,34 +281,39 @@ Parser.prototype._parse = function (node, context, config) {
     element.attribs['no-close'] = 'true';
     element.attribs['no-switch'] = 'true';
     element.name = 'panel';
+    delete element.attribs.boilerplate;
     break;
   }
   case 'morph': {
```

Comment:
I think eventually we might deprecate this element. This block of logic duplicates with `panel` one.

Reply:
Do I keep the code and only do it here (and not in preprocess)?
Also, should we poss the `logger` in the options or use `console.warn`?

Reply:
Yes, keep the code because we should still make it correct.
Only doing it here is fine, we don't need repeated warnings.
Oh, `logger` is in cli. `console.warn` will suffice.